salesforce/9.8/modules/ROOT/pages/salesforce-connector-studio.adoc

```diff
@@ -232,21 +232,51 @@ image::salesforce-mutual-tls.png[Mutual TLS]
 
 Salesforce uses the same session for all your threads, so for example, if your session is active and you log in again, Salesforce uses the existing session instead of creating a new one.
 
-If the *Disable session invalidation* field is set to False, the connector automatically destroys the session after it's no longer needed.
+If the *Disable session invalidation* field is set to `false`, the connector automatically destroys the session after it's no longer needed.
 
-You should keep the session alive when you are working with threads or concurrency in general. Salesforce uses the same session for all your threads (for example, if you have an active session and you log in again, Salesforce uses the existing session instead of creating a new one). To make sure the connection doesn't close when a thread is finished, you should set the *Disable session invalidation* field to True in the Connection section of the connector's global element properties.
+You should keep the session alive when you are working with threads or concurrency in general. Salesforce uses the same session for all your threads (for example, if you have an active session and you log in again, Salesforce uses the existing session instead of creating a new one). To make sure the connection doesn't close when a thread is finished, you should set the *Disable session invalidation* field to `true` in the *Connection* section of the connector's global element properties.
 
 image::salesforce-disable-session.png[Disable Session Field]
 
+[[keepalive]]
+== Keep Alive Feature (Deprecated in Salesforce 10.x)
+
+To keep a session alive, you can set a scheduler to make one simple API request (get server timestamp), within a specified timeframe.
+To activate this feature, you must configure the *Advanced reconnection params* in the Salesforce Config *General* tab, which include:
+
+* Maintain session request frequency +
+The default value is 60 seconds
+* Initial delay +
+The default value is 480 seconds
+
+If no values are passed by the user, the default values are used.
+These parameters are configurable because every Salesforce instance can specify a session timeout value, which can vary from 15 minutes to 24 hours.
+
+=== Example
+
+If the session timeout on the instance is set to 15 minutes and the application does not make any requests to Salesforce for 15 minutes or more, the session expires; and if the reconnection parameter is not enabled, subsequent API requests fail. 
+
+In order to keep the session alive, you can configure the *Advanced reconnection params* like this:
+
+.	Initial delay +
+It is best to set this value at lower than 15, for example, 13 minutes. +
+This value can also be set at a higher value than the session timeout value if it is known that the application will make requests to Salesforce and will not let the session expire, but it is best to set it for a lower value.
```

Comment:
I suggest: 

You can also set this value to a higher value than the session timeout value if you know that the application will make requests to Salesforce and not let the session expire, but it is best to set it to a lower value.